profiler/hotmid/main.go

```diff
@@ -0,0 +1,118 @@
+// Copyright 2018 Google Inc. All rights reserved.
+// Use of this source code is governed by the Apache 2.0
+// license that can be found in the LICENSE file.
+
+// [START hotmid]
+
+// Sample hotmid is an application that simulates multiple calls to a library
+// function made via different call paths. Each of these calls is not
+// particularly expensive (and so does not stand out on the flame graph). But
+// in the aggregate these calls add up to a significant time which can be
+// identified via looking at the flat list of functions' self and total time.
+package main
+
+import (
+	"flag"
+	"log"
+	"time"
+
+	"cloud.google.com/go/profiler"
+)
+
+var (
+	// Service version to configure.
```

Comment:
Love these comments. Would you mind starting them with the thing being described? https://github.com/golang/go/wiki/CodeReviewComments#comment-sentences

Reply:
Done.